downgrade

```diff
@@ -96,17 +96,17 @@ prompt_to_ignore() {
     if [[ "${ans,,}" == $(gettext 'y')* ]]; then
       ln="$(grep -n -m 1 '^ *IgnorePkg' "$PACMAN_CONF" | cut -d : -f 1)"
       if [ -n "$ln" ]; then
-        sudo sed -i "$ln s/.*/& $pkg/" "$PACMAN_CONF"
+        sed -i "$ln s/.*/& $pkg/" "$PACMAN_CONF"
         continue
       fi
 
       ln="$(grep -n '^# *IgnorePkg' "$PACMAN_CONF" | tail -n 1 | cut -d : -f 1)"
       if [ -n "$ln" ]; then
-        sudo sed -i "$ln s/.*/&\\nIgnorePkg = $pkg/" "$PACMAN_CONF"
+        sed -i "$ln s/.*/&\\nIgnorePkg = $pkg/" "$PACMAN_CONF"
         continue
       fi
 
-      printf "IgnorePkg = %s\\n" "$pkg" | sudo tee -a "$PACMAN_CONF"
+      printf "IgnorePkg = %s\\n" "$pkg" | tee -a "$PACMAN_CONF"
```

Comment:
```suggestion
      printf "IgnorePkg = %s\\n" "$pkg" >> "$PACMAN_CONF"
```

I don't feel strongly, but this would work now since the shell itself has privileges now.

Reply:
Will change.